### PR TITLE
Add learning resources and links for shell scripting (#4)

### LINKED ISSUE #4: Add resources to get started with scripting
Update README.md with resources (links) to get basic knowledge about shell scripting to automate tasks in Linux.
Focus on the quality and relevance of those resources.

### PR DESCRIPTION
**Pull request by Piyush-Chetwani**
Added learning resources and links to get basic knowledge about shell scripting to automate tasks in Linux in the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,35 @@ A set of recipes for automating Linux tasks.
 Want me to write something to automate your Linux tasks?
 
 I would love to do so. Just open up an issue in the issue tracker.
+
+
+
+# Resources
+
+## Beginners (to learn shell scripting)
```
Comment (aquatiko) [inline README.md:29]: Good work with the resources. I would love to see some good youtube playlist links in this section as well.

**Comment by aquatiko** [comment]
Also, can you update the title of this PR to something which suits better? 

Bonus points: 
Can you change the commit message to something better reflecting the change? Will require you to use a commit rename feature from the git terminal.

File: README.md (hunk 1 above)
Comment (Piyush-Chetwani) [reply README.md:29]: Ok sir, will add some YouTube playlist as well and would try to change the name of the commit using terminal.

**Comment by aquatiko** [comment]
@Piyush-Chetwani Good work with the exhaustive list of resources.

**Pull request merged**